renderdoc/os/posix/posix_stringio.cpp

```diff
@@ -83,6 +83,11 @@ void CreateParentDirectory(const string &filename)
   }
 }
 
+bool IsRelativePath(const string &path)
+{
+  return path.front() != '/';
```

Comment:
Good catch, thanks.  I went with "empty is not a path" because the code is clean, but could be convinced otherwise.